src/app/screens/GenerateAccount/GenerateAccountEnd.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import html2canvas from 'html2canvas';
+
+import {
+  Address,
+  Body,
+  Box,
+  Button,
+  Heading,
+  IFlowComponentProps,
+  Link,
+  Panel,
+  PaperWallet
+} from '@components';
+import { getKBHelpArticle, KB_HELP_ARTICLE } from '@config/helpArticles';
+import { DEFAULT_DERIVATION_PATH, getGeneratedAccount, useSelector } from '@store';
+import { translate, translateRaw } from '@translations';
+
+export const GenerateAccountEnd = ({ onNext }: IFlowComponentProps) => {
+  const paperWallet = useRef<HTMLDivElement>();
+  const [showMnemonicPhrase, setShowMnemonicPhrase] = useState(false);
+  const [paperWalletImage, setPaperWalletImage] = useState<string>();
+  const { address, mnemonicPhrase } = useSelector(getGeneratedAccount);
+
+  const handleClick = () => setShowMnemonicPhrase(true);
+
+  useEffect(() => {
+    if (paperWallet.current) {
+      // Timeout is required to make sure the QR codes are rendered
```

Comment:
Why?

Reply:
The QR codes are rendered async, without the timeout it would generate an image without QR codes.